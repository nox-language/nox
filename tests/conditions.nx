let var num := 42
in (
    if num > 40 then
        print("num > 40\n")
    else
        print("num <= 40\n");

    if num < 40 then
        print("num < 40\n")
    else
        print("num >= 40\n");

    if num > 42 then
        print("num > 42\n")
    else
        print("num <= 42\n");

    if num < 42 then
        print("num < 42\n")
    else
        print("num >= 42\n")
)
end
