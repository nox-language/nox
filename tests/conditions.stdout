num > 40
num >= 40
num <= 42
num >= 42
